src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+
+  display: flex;
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star-active.svg);
+
+  display: flex;
+}
```

Comment:
The `.stars__star` class already defines the `width`, `height`, `background-repeat`, `background-position`, and `display` properties. Redefining these properties for each `.stars--X .stars__star:nth-child(-n + X)` selector is unnecessary and repetitive. Consider removing the duplicated properties to keep your CSS DRY (Don't Repeat Yourself).